src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1409,4 +1409,7 @@ This version used in: {2}</value>
   <data name="Use_expression_body_for_lambda_expressions" xml:space="preserve">
     <value>Use expression body for lambda expressions</value>
   </data>
+  <data name="Convert_to_linq_call_form" xml:space="preserve">
+    <value>Convert to LINQ (call form)</value>
```

Comment:
> `<value>Convert to LINQ (call form)</value>`

I assume this is part of the merge, not a change in this PR. I thought the github diff view handled this better...

Reply:
Confirmed this doesn't show in diff for entire PR. #Closed